inc/namespace.php

```diff
@@ -107,10 +107,27 @@ function register_asset( string $manifest_path, string $target_asset, array $opt
 	$asset_handle = $options['handle'] ?? $target_asset;
 	$asset_version = Manifest\get_version( $asset_uri, $manifest_path );
 
+	// If running the development build with runtimeChunk: single, a runtime file will be present in the manifest.
+	// Register this and ensure it is loaded only once per page.
```

Comment:
```suggestion
	// If running the development build with runtimeChunk: single, a runtime
	// file will be present in the manifest. Register this and ensure it is
	// loaded only once per page.
```
Nitpick, comment line length